pages/api/mail/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ResponseUtil } from "@utils/responseUtil";
+import type { NextApiRequest, NextApiResponse } from "next";
+import send from "@utils/sendMail";
+import findEmails from "@database/mail";
+
+/**
+ * Handles the request to /api/mail (POST)
+ * @param req request object
+ * @param res response object
+ * @returns Promise<void> and sends json object back to the client (type: ResponseObject)
+ */
+export default async function mailHandler(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "POST" && req.body.key == process.env.AWS_LAMBDA_KEY) {
+        // AWS_LAMBDA_SECRET
+        const result3h = await findEmails(3);
+        const result48h = await findEmails(48);
+        for (let i = 0; i < result3h.length; ++i) {
```

Comment:
Any particular reason we're using `for` loops instead of `foreach`?

Reply:
I'm not sure how to use foreach with the indexing of the parentRegs and volunteerRegs